### DISCUSSION TITLE
GitHub Coding Project on Mobile

### DISCUSSION THREAD

**Discussion by ImpenetrableNoble** [community/community#121114] [Mobile]
How come I can't use the mobile application of GitHub to start coding my own project? I think it'll be easier that way rather than just doing it within the desktop website, it's unfair and ridiculous that no developer of GitHub is not taking any notice of this at all.

**Comment by sl1taz** [comment]
Github app doesn't have this feature yet.
Maybe you can access `https://vscode.dev` and signin with your Github account and use codespaces inside vscode on mobile browser.

Or, you can access your Github repo 
`https://github.com/username/your_repo`

And change github.com to github.dev
`https://github.dev/username/your_repo`

This will load Github Web IDE on your browser with your repositorie.

**Reply by Tyotegc** [reply]
Hi
